assets/js/blocks/cart-checkout/mini-cart/block.tsx

```diff
@@ -154,6 +167,11 @@ const MiniCartBlock = ( {
 						getCurrencyFromPriceResponse( cartTotals )
 					) }
 				</span>
+				{ labelTaxName !== '' && subTotal !== 0 && (
+					<small className="wc-block-mini-cart__tax_label">
```

Comment:
[We try to use BEM for CSS class names](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/docs/contributors/coding-guidelines.md#naming), so `_` should only be used to separate parts of the selector. When it's the same part but different words, we use `-`:
```suggestion
					<small className="wc-block-mini-cart__tax-label">
```